src/Microsoft.Diagnostics.ExtensionCommands/Host/RuntimesCommand.cs

```diff
@@ -41,9 +41,9 @@ public override void Invoke()
             }
             else
             {
-                foreach (IRuntime runtime in RuntimeService.Runtimes)
+                foreach (IRuntime runtime in RuntimeService.EnumerateRuntimes())
                 {
-                    string current = RuntimeService.Runtimes.Count() > 1 ? runtime == RuntimeService.CurrentRuntime ? "*" : " " : "";
+                    string current = RuntimeService.EnumerateRuntimes().Count() > 1 ? runtime == RuntimeService.CurrentRuntime ? "*" : " " : "";
```

Comment:
I should move the .Count() out of the loop.  And yes this code purposely doesn't add a star if there is only one runtime.  Since one runtime is the usual case I felt that always adding the star confuses/makes unclear this command's output.